.github/workflows/tests.yml

```diff
@@ -436,15 +436,22 @@ jobs:
             vscode "github.copilot" if OS.mac?
           EOS
 
+          set +x
```

Comment:
```suggestion
```
This isn't needed because of

https://github.com/Homebrew/brew/blob/bf5bfd5a22b70d1b526de513015034408006e00e/.github/workflows/tests.yml#L20-L22

See an earlier run without `set +x` here: https://github.com/Homebrew/brew/actions/runs/14328445005/job/40158768563?pr=19722#step:5:1